frontend/src/components/MonacoEditor/AutoCompletion.ts

```diff
@@ -40,7 +40,8 @@ export default class AutoCompletion {
     return range;
   }
 
-  getCompletionItemsForKeywords() {
+  async getCompletionItemsForKeywords() {
+    const monaco = await import("monaco-editor");
```

Comment:
It looks like the methods in `AutoCompletion` will be called many times. So would dynamic import be better than static import?

Reply:
I guess that will be optimized by the module loader